tsp-output/@azure-tools/typespec-csharp/src/Generated/Models/AudioSegment.Serialization.cs

```diff
@@ -205,15 +205,15 @@ AudioSegment IPersistableModel<AudioSegment>.Create(BinaryData data, ModelReader
         string IPersistableModel<AudioSegment>.GetFormatFromOptions(ModelReaderWriterOptions options) => "J";
 
         /// <summary> Deserializes the model from a raw response. </summary>
-        /// <param name="response"> The result to deserialize the model from. </param>
+        /// <param name="response"> The response to deserialize the model from. </param>
```

Comment:
These changes are low-priority since the APIs are internal, but if they are easy, they would make the code more readable and correct.

Reply:
https://github.com/Azure/autorest.csharp/issues/4486